src/program_automation.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+""" Automatically prioritize/rank JIRA stories attached to a JIRA feature
```

Comment:
That doc string does not seem to be correct.

Reply:
Updated in 4eb8856.